nautobot/core/api/schema.py

```diff
@@ -401,3 +417,20 @@ def map_serializer_field(self, auto_schema, direction):
         On requests, require PK only; on responses, represent the entire nested serializer.
         """
         return auto_schema._map_serializer(self.target.serializer, direction)
+
+
+class EmailFieldFix(OpenApiSerializerFieldExtension):
+    """Fix invalid schema for EmailField when default is an empty string."""
+
+    target_class = "rest_framework.serializers.EmailField"
+
+    def map_serializer_field(self, auto_schema, direction):
+        if self.target.default == "":
+            return {
+                "oneOf": [
+                    {"type": "string"},
+                    {"type": "string", "format": "email"},
```

Comment:
Can/should we still use `build_basic_type(OpenApiTypes.EMAIL)` here in some fashion? I note we lost the `maxLength` constraint in the updated schema, not sure how important/relevant that is.

Reply:
Should be fixed in the last commit